data/pigui/libs/ship-equipment.lua

```diff
@@ -26,196 +28,108 @@ local iconSize = Vector2(pionillium.body.size)
 
 local equipmentInfoTab
 
----@class EquipmentWidget
----@field meta table
-local EquipmentWidget = utils.inherits(nil, "EquipmentWidget")
-
--- Slot information for the empty slot + example slot data layout
-local emptySlot = {
-	-- type = "Slot", name = "[EMPTY]"
-	icon = icons.autopilot_dock, --size = "S1",
-	{ icons.hull, ui.Format.Mass(0, 1), le.EQUIPMENT_WEIGHT },
-	-- { icons.repairs, "100%", le.EQUIPMENT_INTEGRITY }
-	-- { icons.ecm_advanced, "0 KW", "Max Power Draw" },
-	-- { icons.temperature, "0 KW", "Operating Heat" },
-}
-
--- Equipment item grouping by underlying slot type
--- TODO: significant refactor to slot system to reduce highly-specialized slots
-local sections = {
-	{ name = le.PROPULSION, slot = "engine", showCapacity = true },
-	{ name = le.WEAPONS, slot = "laser_front", showCapacity = true },
-	{ name = le.MISSILES, slot = "missile", showCapacity = true },
-	{ name = le.SCOOPS, slot = "scoop", showCapacity = true },
-	{ name = le.SENSORS, showCapacity = true, slots = {
-		"sensor", "radar", "target_scanner", "hypercloud"
-	} },
-	{ name = le.SHIELDS, slot = "shield" },
-	{ name = le.UTILITY, slots = {
-		"cabin", "ecm", "hull_autorepair",
-		"energy_booster", "atmo_shield",
-		"laser_cooler", "cargo_life_support",
-		"autopilot", "trade_computer", "thruster"
-	} }
+---@class UI.EquipmentWidget : UI.Module
+---@field New fun(id): self
+local EquipmentWidget = utils.class("UI.EquipmentWidget", Module)
+
+-- Equipment item grouping by underlying slot kinds
+EquipmentWidget.Sections = {
+	{ name = le.PROPULSION, types = { "engine", "thruster", "hyperdrive" } },
+	{ name = le.WEAPONS, type = "weapon" },
+	{ name = le.MISSILES, types = { "pylon", "missile_bay", "missile" } },
+	{ name = le.SHIELDS, type = "shield" },
+	{ name = le.SENSORS, type = "sensor", },
+	{ name = le.COMPUTER_MODULES, type = "computer", },
+	{ name = le.CABINS, types = { "cabin" } },
+	{ name = le.HULL_MOUNTS, types = { "hull", "utility" } },
```

Comment:
Fuel scoops are not in any of these categories. They should possibly be in HULL_MOUNTS?

```suggestion
	{ name = le.HULL_MOUNTS, types = { "hull", "utility", "fuel_scoop" } },
```

However, this only partially fixes fuel-scoops, as there is no translation for "HARDPOINT_FUEL" and the Equipment Market isn't offering up any scoops.

Finally, cargo scoops seem to still be missing completely.

Reply:
Actually, it should be :
```suggestion
	{ name = le.HULL_MOUNTS, types = { "hull", "utility", "scoop" } },
```

Along with all ship definitions changing the scoop "type" field from "fuel_scoop" to "scoop.fuel".  This now shows up the fuel scoop slots in the Equipment Market, _and_ allows purchasing a fuel scoop for those slots.

Reply:
I appreciate the catch on this one! The first is "more correct", though I am considering breaking scoops out into their own category within the equipment list.

We decided on IRC to make cargo scoops a non-slot equipment, and give fuel scoops their own dedicated slot - I've just not updated the equipment lists yet to match the changes in ship hardpoints.

Reply:
Updated fuel scoops with placeholder equipment items and proper slot types.